aria-practices.html

```diff
@@ -6424,6 +6424,327 @@ <h3>Indicating sort order with <code>aria-sort</code></h3>
+      </p>
+
+      <pre><code>&lt;label for="loadstatus"&gt;Loading:&lt;/label&gt;
+&lt;progress id="loadstatus" max="100" value="33"&gt;&lt;/progress&gt;
+      </code></pre>
+    </section>
+
+    <section id="range_related_properties_scrollbar_role">
+      <h3>Range properties with scrollbars</h3>
+
+      <p>
+        The <code>aria-valuemin</code> and <code>aria-valuemax</code> properties only need to be set for the <code>scrollbar</code> role when it's minimum value is not 0 or the maximum value is not 100.   The aria-valuenow property is required for <code>scrollbar</code> and the author needs to make sure it is within the minimum and maximum values.
+      </p>
+      <p>
+        <code>aria-valuenow</code> will generally be exposed as a percentage between <code>aria-valuemin</code> and <code>aria-valuemax</code> calculated by assistive technologies.
+      </p>
+
+      <pre><code>&lt;div id="pi"&gt;
+3.1415926535897932384626433832795028841971693993751058209749445923078164062862089986280348253421170679
+&lt;/div&gt;
+&lt;div class="rail"&gt;
+  &lt;div
+    class="thumb"
+    role="scrollbar"
+    //The scrollbar uses the default values of 0 and 100 for aria-valuemin and aria-valuemax
+    aria-controls="pi"
+    aria-orientation="horizontal"
+    aria-valuenow="25"&gt;
+  &lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+
+    </section>
+
+    <section id="range_related_properties_slider_role">
+      <h3>Range properties with sliders</h3>
+      <p>
+        The <code>aria-valuemin</code> and <code>aria-valuemax</code> properties only need to be set for the <code>slider</code> role when the slider's minimum is not 0 or the maximum value is not 100.
+        The <code>aria-valuenow</code> property is required for <code>slider</code> role and the author needs to make sure it is within the minimum and maximum values.
+        Detailed description of the <code>slider</code> role can be found in the <a href="#slider">slider design pattern</a> and <a href="#slidertwothumb">slider (multi-thumb) design pattern</a>.
+      </p>
+      <p>
+        The following example is a temperature controller.
+        <code>aria-valuetext</code> is not be used as the number value in <code>aria-valuenow</code> is meaningful to the user.
+      </p>
+      <pre><code>&lt;div class="rail"&gt;
+  &lt;div id="thumb" role="slider" aria-valuemin="50" aria-valuenow="68" aria-valuemax="100"
+       aria-label="Temperature (F)" tabindex="0"&gt;
+  &lt;/div&gt;
+&lt;/div&gt;</code></pre>
+      <p>The slider example above can be made using the HTML <code>&lt;input type="range"></code> element.</p>
+      <pre><code>&lt;input type="range" min="50" value="68" max="100" aria-label="Temperature (F)"&gt;
+      </code></pre><p>The following example is a fan control. The <code>aria-valuenow</code> value is "1", which is not meaningful to the user. The assistive technology will surface the value of <code>aria-valuetext</code> ("low") instead.</p>
```

Comment:
```suggestion
      </code></pre><p>The following example is a fan control. The <code>aria-valuenow</code> value is "1", which is not meaningful to the user. The <code>aria-valuetext</code> property is used so that assistive technology will surface its value ("low") instead.</p>
```